<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_auidoPath"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_audioName">
        <property name="text">
         <string>No audio file loaded</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>308</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_browse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_load">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_parameters">
        <property name="text">
         <string>Parameters(JSON)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_parameters"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_play">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_stop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="playSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_audioTime">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>00:00:00/00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_log"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAudio_Play">
    <property name="title">
     <string>Operations</string>
    </property>
    <widget class="QMenu" name="menuVolume">
     <property name="title">
      <string>Volume</string>
     </property>
     <addaction name="actionAdjustByJson"/>
     <addaction name="actionNormalization"/>
    </widget>
    <addaction name="actionReverse"/>
    <addaction name="actionFade"/>
    <addaction name="menuVolume"/>
    <addaction name="actionResample"/>
    <addaction name="actionOLA"/>
    <addaction name="actionFFT"/>
   </widget>
   <addaction name="menuAudio_Play"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReverse">
   <property name="text">
    <string>Reverse</string>
   </property>
   <property name="toolTip">
    <string>Reverse</string>
   </property>
  </action>
  <action name="actionFade">
   <property name="text">
    <string>Fade</string>
   </property>
  </action>
  <action name="actionResample">
   <property name="text">
    <string>Resample</string>
   </property>
  </action>
  <action name="actionOLA">
   <property name="text">
    <string>OLA</string>
   </property>
  </action>
  <action name="actionFFT">
   <property name="text">
    <string>FFT</string>
   </property>
  </action>
  <action name="actionAdjustByJson">
   <property name="text">
    <string>Adjust By Json</string>
   </property>
  </action>
  <action name="actionNormalization">
   <property name="text">
    <string>Normalization</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
